glyphs-reader/data/update.py

```diff
@@ -13,16 +13,60 @@
 from dataclasses import dataclass
 import glyphsLib
 from importlib import resources
-from io import StringIO
 from lxml import etree
 from pathlib import Path
 from textwrap import dedent
 from typing import Iterable, Mapping, Optional, Tuple
 
 
+_CATEGORIES = (
+    "Mark",
+    "Space",
+    "Separator",
+    "Letter",
+    "Number",
+    "Symbol",
+    "Punctuation",
+    "Other",
+)
+
+_SUBCATEGORIES = (
+    "Spacing",
+    "Radical",
+    "Math",
+    "Superscript",
+    "Geometry",
+    "Dash",
+    "DecimalDigit",
+    "Currency",
+    "Fraction",
+    "Halfform",
+    "Small",
+    "Number",
+    "Quote",
+    "Space",
+    "Letter",
+    "Jamo",
+    "Format",
+    "Parenthesis",
+    "Matra",
+    "Arrow",
+    "Nonspacing",
+    "Compatibility",
+    "Syllable",
+    "Ligature",
+    "Modifier",
+    "SpacingCombining",
+    "Emoji",
+    "Enclosing",
+    "Composition",
+    "Other",
```

Comment:
okay so actually this is just copied from the existing enum? The enum impl may change, and I wouldn't want the source of truth to be this python file..

Reply:
> I have reason to suspect we might need to rework the implementation of these enums in the future to account for arbitrary string values, and that will be significantly more annoying if they are defined in some python codegen file

> I wouldn't want the source of truth to be this python file.

Python producing code seems to be the sole point of contention, I have less concern about this perhaps. Will followup in IM.

Reply:
Changed per IM, avoids the key concern about fragility